main.cpp

```diff
@@ -190,27 +266,37 @@ Thing 2) Audio Compressor Limiter Effect
 
 struct AudioCompLimiterEffect
 {
-    // Threshold for Compressor
     float compThreshold = -6.f;
-    // Make Up Gain for Compressor
     float compMakeupGain = +3.f;
-    // Threshold for Limiter
     float limiterThreshold = -3.f;
-    // Amount of gain change
     float gainChange = 0.f;
-    // Compression ratio
     float compRatio = 3.f;
 
-    // Compress Audio
-    void compressAudio(float compRatio, float compThreshold);
-    // Limit Audio
+    void compressAudio(float compressionRatio, float compressionThreshold);
     void limitAudio();
-    // Change Audio Gain
-    void changeGain(float gainChange);
+    void changeGain(float newGain);
 
 
 };
 
+void AudioCompLimiterEffect::compressAudio(float compressionRatio, float compressionThreshold)
+{
+    this->compRatio = compressionRatio;
+    this->compThreshold = compressionThreshold;
+}
```

Comment:
Don't use concepts that have not been covered yet, like the `this` keyword. 
Also, that keyword is not needed, because this is a member function (you can learn about `hidden this` later if you choose to research the topic or continue with the full course)
- [ ] revise